eventsub-common/src/main/java/com/github/twitch4j/eventsub/domain/chat/Announcement.java

```diff
@@ -0,0 +1,16 @@
+package com.github.twitch4j.eventsub.domain.chat;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Setter;
+
+@Data
+@Setter(AccessLevel.PRIVATE)
+public class Announcement {
+
+    /**
+     * Color of the announcement.
+     */
+    private String color;
```

Comment:
Should we copy the helix enum here? technically eventsub doesn't document the possible values for `color`, but there's no reason it should differ